### ISSUE TITLE
Fast Pool can't be used without wrapper contract

### ISSUE THREAD

**Issue by friedger** [hirosystems/btcweb3#29]
Normally, users can use the Fast pool contract as pool address in custom pool. 
This is broken due to https://github.com/hirosystems/stacks.js/issues/1456

Pressing confirm and start pooling does not, only an error log in the console
```
Uncaught (in promise) Error: Not a c32-encoded string
    at c32decode (encoding.js:91:15)
    at c32checkDecode (checksum.js:62:36)
    at c32addressDecode (address.js:57:27)
    at createAddress (postcondition-types.ts:114:23)
    at standardPrincipalCV (principalCV.ts:58:16)
    at StackingClient.getDelegateOptions (index.ts:1056:9)
    at getOptions (utils-delegate-stx.ts? [sm]:95:19)
    at handleSubmit (utils-delegate-stx.ts? [sm]:153:32)
    ```

**Comment by markmhx** [comment]
Will this require anything else on the app's end aside from integrating a new version of stacks.js?

**Comment by friedger** [comment]
No, a new version should fix this.

A workaround would be to not use stacks.js for this function call.

**Comment by friedger** [comment]
addressed in 130d2171d1891eeaa153d8e94010090d4e727051
